webapp/app.py

```diff
@@ -217,6 +217,62 @@ def start_career():
     return flask.render_template("/careers/career-explorer.html")
 
 
+@app.route("/careers/progression")
+def careers_progression():
+    get_vacancies = greenhouse.get_vacancies()
+    all_departments = (_group_by_department(get_vacancies),)
+
+    dept_list = [
```

Comment:
Is this logic now duplicated? If so, we should centralise it with a reuseable util.

Reply:
Ok I think I found a way to achieve both.